x/datapool/types/keys.go

```diff
@@ -54,11 +61,15 @@ func GetKeyPrefixDataValidateCert(poolID, round uint64, dataHash []byte) []byte
 	return append(GetKeyPrefixDataValidateCertByRound(poolID, round), dataHash...)
 }
 
-func GetKeyPrefixNFTRedeemReceiptByPoolID(poolID uint64, redeemer sdk.AccAddress) []byte {
-	keyPoolAppended := append(KeyPrefixNFTRedeemReceipts, sdk.Uint64ToBigEndian(poolID)...)
-	return append(keyPoolAppended, redeemer.Bytes()...)
+func GetKeyPrefixNFTRedeemReceiptByPoolID(poolID uint64) []byte {
+	return append(KeyPrefixNFTRedeemReceipts, sdk.Uint64ToBigEndian(poolID)...)
+}
+
+func GetKeyPrefixNFTRedeemReceipt(poolID, round, nftID uint64) []byte {
+	combineKeys := CombineKeys(GetKeyPrefixNFTRedeemReceiptByPoolID(poolID), sdk.Uint64ToBigEndian(round))
+	return append(KeyPrefixNFTRedeemReceipts, CombineKeys(combineKeys, sdk.Uint64ToBigEndian(nftID))...)
```

Comment:
The `KeyPrefixNFTRedeemReceipts` is included In `combineKeys`, because `GetKeyPrefixNFTRedeemReceiptByPoolID()` includes it. 
```suggestion
func GetKeyPrefixNFTRedeemReceipt(poolID, round, nftID uint64) []byte {
	combineKeys := CombineKeys(GetKeyPrefixNFTRedeemReceiptByPoolID(poolID), sdk.Uint64ToBigEndian(round))
	return CombineKeys(combineKeys, sdk.Uint64ToBigEndian(nftID))
```

Reply:
Thanks. Fixed in [0257606](https://github.com/medibloc/panacea-core/pull/322/commits/025760608166234f18903cea8994753770da6045).